dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityOperationParamsMapper.java

```diff
@@ -288,9 +288,9 @@ private void validateGlobalSearchParameters(TrackedEntityQueryParams params, Use
         validateSearchableAttributes(params, searchableAttributeIds);
       }
 
-      int maxTeiLimit = getMaxTeiLimit(params);
-      checkIfMaxTeiLimitIsReached(params, maxTeiLimit);
-      params.setMaxTeLimit(maxTeiLimit);
+      int maxTeLimit = getMaxTeLimit(params);
+      checkIfMaxTeLimitIsReached(params, maxTeLimit);
+      params.setMaxTeLimit(maxTeLimit);
```

Comment:
If you move this like one up, then you don't need to send the `maxTeLimit` as a parameter to the `checkIfMaxTeLimitIsReached` function anymore, as it is already contained in the `params` parameter.